backend/contributions/tests/test_community_awards.py

```
@@ -0,0 +1,442 @@
+            [
+                ('invalid_format', 1),
+                ('invalid_amount', 3),
+                ('invalid_amount', 4),
+                ('invalid_amount', 5),
+                ('ok', 6),
+            ],
+        )
+
+    def test_row_cap(self):
+        text = '\n'.join(f'user{i} 10' for i in range(301))
+        with self.assertRaises(Exception):
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)